### ISSUE TITLE
RFE: 'Solo' mode

### ISSUE THREAD

**Issue by Baylink-gh** [IJMacD/newsmap-js#24]
I see this was mentioned in your reddit thread when you announced, but the commenter wasn't clear.  And just having found your delightful reimplementation of newsmap, I wanted to suggest it:

When all categories are selected, and I click on one of the tabs, instead of *deselecting* that tab, select *only* that tab.

If only one tab is selected and the user clicks it, then reselect all tabs.  (A different way to put this is "if the user deselects the only active tab, and none are selected, automatically force the selection set back to All".)

Audio consoles (and things which have stolen it from them) refer to this as solo mode: you click one button, and that's all you hear.  If you click it a second time, and it's the only one selected, you go back to hearing everything in the mix.

Secondary question: I didn't see an "autorefresh time" in the Settings; does the page actually refresh automatically?

**Comment by IJMacD** [comment]
Thanks for getting in touch. At the moment clicking the categories toggles them on/off as you describe. Alt-Click will achieve what I think you are referring to as Solo Mode. I admit this feature certainly isn't very discoverable. I could add an option to reverse the default behaviour to be in line with what you describe.

As for the second part of the issue - yes it does auto-refresh. That time is hard coded in App.js#L58 to be 10 minutes. Again I could extract this to the settings to make it configurable.

Of course if you feel like implementing any of these I'll happily take a look at any pull requests you send.

**Comment by Baylink-gh** [comment]
Yeah, I know.  I'll take a look at it.  FWIW, though, in a current Firefox 64, Alt-Click doesn't seem to do anything.

And no, 10 minutes is just fine.  You just did such a nice job on the hamster dance... :-)

**Comment by Baylink-gh** [comment]
Nope, I've tried it in a couple browsers; Alt-Click does not seem to Solo a specific category on any of them.

Related: if I *manually* solo one or two categories by turning the rest off, it scales the *size*, rather than the article count, which would be the result I'd prefer; should I open a separate ticket on that, or nah?  :-)
